db/migrate/20191002224831_add_category_to_institutions.rb

```diff
@@ -0,0 +1,5 @@
+class AddCategoryToInstitutions < ActiveRecord::Migration[5.0]
+  def change
+    add_column :institutions, :category, :integer, null: false, default: 0
```

Comment:
This should maybe not have a default value and instead just update all existing institutions to be universities, thoughts?